packages/api/internal/dns/server.go

```diff
@@ -9,14 +9,24 @@ import (
 	"sync"
 	"time"
 
-	"github.com/go-redis/cache/v8"
-	"github.com/go-redis/redis/v8"
+	"github.com/go-redis/cache/v9"
 	resolver "github.com/miekg/dns"
+	"github.com/redis/go-redis/v9"
 	"go.uber.org/zap"
 
 	"github.com/e2b-dev/infra/packages/shared/pkg/smap"
 )
 
+// Rediser generalize RedisClient and RedisClusterClient, taken from github.com/go-redis/cache/v9
```

Comment:
After migration period, we will be using only one of them, this is prep for smoother transition. There will be only one client afterwards